src/Compilers/VisualBasic/Portable/Symbols/Symbol_Attributes.vb

```diff
@@ -192,6 +192,10 @@ Namespace Microsoft.CodeAnalysis.VisualBasic
             Dim compilation = Me.DeclaringCompilation
             MarkEmbeddedAttributeTypeReference(arguments.Attribute, arguments.AttributeSyntaxOpt, compilation)
             ReportExtensionAttributeUseSiteError(arguments.Attribute, arguments.AttributeSyntaxOpt, compilation, arguments.Diagnostics)
+
+            If arguments.Attribute.IsTargetAttribute(Me, AttributeDescription.SkipLocalsInitAttribute) Then
+                arguments.Diagnostics.Add(ERRID.WRN_AttributeNotSupportedInVB, arguments.AttributeSyntaxOpt.Location, arguments.Attribute.ToString())
```

Comment:
>arguments.Attribute.ToString() [](start = 118, length = 30)

It doesn't look like we use this for error reporting anywhere else and this function does too much. Consider using AttributeDescription.SkipLocalsInitAttribute.FullName instead.

Reply:
Updated it.